tests/test_flows.py

```diff
@@ -112,6 +113,14 @@ def tcpdump_start(self):
             else:
                 self.err("tcpdump terminated")
                 self.tcpdump_proc = None
+            # after tcpdump finishes, add a list of all ieee1905 packets for easier analysis
+            # first, generate a json file of the pcap dump
+            self.json_from_pcap = subprocess.run(['tshark', '-r', outputfile, '-T', 'json' '>tmp.json'])
+            if self.json_from_pcap.returncode == 0:
+                with open('tmp.json', 'r') as f:
+                    #filter out only relevant packets from the file
+                   ieee1905_packets =  filter(lambda x: "ieee1905" in x["_source"]["layers"],json.load(f))
```

Comment:
We can also filter at the tcpdump level, that's probably more efficient...

While we're at it, we should also add the `-U` option to tcpdump to make sure the pcap file is always up to date.

Reply:
I've tried adding a `-j` filter for ieee1905 messages but it was way too strict and resulted in a huge loss of information.

Reply:
`-j` filter? I mean a simple pcap-filter expression when doing the tcpdump capture, something like `ether proto 0xwhateveritis`.

Speaking of which: even simpler would be to replace tcpdump with tshark directly, using a combination of `-w` and `-P` to both write to a pcap file and print to stdout. We can then just read from the `self.tcpdump_proc.stdout` and process in the middle of the test.

BTW, this PR is definitely going in the right direction :-)

Reply:
+1 for using tshark directly